Assets/Resources.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
What is that file?

Reply:
stupid file that gets generated for my local projects Resources folder. I have to delete it 100 times an hour. It slipped in this time :(